draft-ietf-quic-recovery.md

```diff
@@ -999,6 +996,18 @@ paces the sending of any packets in excess of the initial congestion window.
 A sender MAY implement alternate mechanisms to update its congestion window
 after idle periods, such as those proposed for TCP in {{?RFC7661}}.
 
+## Application Limited Sending
+
+The congestion window should not be increased in slow start or congestion
+avoidance when it is not fully utilized.  The congestion window could be
+under-utilized due to insufficient application data to send or flow control
+limits.
+
+When the sender is pacing(see {{pacing}}) packets, the sender may be unable
+to use the full congestion window for a period of time after receiving an
+ACK, due to pacing.  In this case, the sender should not consider themselves
```

Comment:
```suggestion
acknowledgment.  In this case, the sender should not consider themselves
```

Reply:
I think I mean ACK, as in the frame?

Reply:
You need more than an ACK, you need one that contains fresh acknowledgments, right?

Reply:
True, changed to acknowledgements.